### PR TITLE
auth dialog - enable context isolation

### PR DESCRIPTION
**Pull request by bpasero**
I think it would be safe to enable `contextIsolation` now that [`preload.ts` conditionally uses `contextBridge`](https://github.com/microsoft/vscode/blob/e5b3ff76bae1fc69df3b94da1d4ac4fe9dda8283/src/vs/base/parts/sandbox/electron-browser/preload.js#L118-L127) if it detects that it runs context isolated.

//cc @joaomoreno 

PS: quite surprisingly we have no NLS story for this dialog, otherwise we could not enable sandbox yet.

### CODE REVIEW

File: src/vs/code/electron-main/auth.ts [hunk 1]
```diff
@@ -60,10 +60,12 @@ export class ProxyAuthHandler extends Disposable {
 			title: 'VS Code',
 			webPreferences: {
 				preload: URI.parse(require.toUrl('vs/base/parts/sandbox/electron-browser/preload.js')).fsPath,
-				enableWebSQL: false,
 				sandbox: true,
-				devTools: false,
+				contextIsolation: true,
+				enableWebSQL: false,
 				enableRemoteModule: false,
+				nativeWindowOpen: true,
```
Comment (deepak1556) [inline src/vs/code/electron-main/auth.ts:67]: `nativeWindowOpen` is not needed when `sandbox: true`, since that is the default behavior in this mode.

Comment (bpasero) [reply src/vs/code/electron-main/auth.ts]: Ok pushed a change to remove it.

**Comment by bpasero** [comment]
Ready for review again.

**Review by deepak1556 — APPROVED** [review]
Manually verified the auth flow continues to work.

**Pull request merged**